challengeutils/__main__.py

```diff
@@ -734,6 +764,47 @@ def build_parser():
     )
     parser_validate_docker.set_defaults(func=command_validate_docker)
 
+    parser_pull_wiki = subparsers.add_parser(
+        'pull-wiki',
+        help='Download a Synapse wiki into markdown and wiki_config.json'
+    )
+    parser_pull_wiki.add_argument(
+        "projectid", type=str,
+        help='Synapse id of Project'
+    )
+    parser_pull_wiki.add_argument(
+        "--workdir", type=str, default="./",
+        help='Path to download markdown files and wiki_config.json.'
+             'Defaults to location of where code is being executed.'
+    )
+    parser_pull_wiki.set_defaults(func=command_pull_wiki)
+
+    parser_validate_wiki_config = subparsers.add_parser(
+        'validate-wiki-config',
+        help='Validate a wiki configuration generated by pull-wiki.')
+    parser_validate_wiki_config.add_argument(
+        "workdir", type=str,
+        help='Path to markdown files and wiki_config.json.'
+    )
+    parser_validate_wiki_config.set_defaults(
+        func=command_validate_wiki_config
+    )
+
+    parser_push_wiki = subparsers.add_parser(
+        'push-wiki',
+        help='Push a Synapse wiki'
+    )
+    parser_push_wiki.add_argument(
+        "projectid", type=str,
```

Comment:
Thanks - I think all of the `*id` variables in this project are named like `projectid`, `wikiid`, `submissionid`. It would be a pretty big shift to change to `project_id`, etc.